src/js/smooch.jsx

```diff
@@ -161,6 +164,14 @@ export class Smooch {
                 }
             });
         }).then((loginResponse) => {
+            observable.on('socket:connected', () => {
```

Comment:
Also what's the best way to notify Smooch from Faye if not through this hub? Eventually we want to listen to socket:closed to show an error message in the UI while Faye is disconnected.